RetroBar/PropertiesWindow.xaml.cs

```diff
@@ -277,6 +277,30 @@ private void AutoStartCheckBox_OnChecked(object sender, RoutedEventArgs e)
         private void cboLanguageSelect_SelectionChanged(object sender, System.Windows.Controls.SelectionChangedEventArgs e)
         {
             LoadVersion();
+
+            if (!EnvironmentHelper.IsWindows10OrBetter)
+            {
+                PopulateClockActionsForPreWindows10();
+            }
```

Comment:
Moving it...

Reply:
Looks like it is still not in the constructor?

Reply:
Oh, I put it in the constructor at first, but I left it here because it runs anyway after the Properties window opens (it's the event that loads when the language value in the combobox is set, and UI related anyway, so I thought it might still be there), it also keeps it consistent when switching languages, so it still runs and updates its values for the clock selection! (Otherwise, please send a commit for this that might suit you better).

Reply:
Seems to be as redundant as `LoadVersion()`, so I'll remove it from the constructor.